packages/core/database/lib/entity-manager/relations-orderer.js

```diff
@@ -33,14 +33,14 @@ const _ = require('lodash/fp');
  * @param {number} [maxOrder=0] - Used to calculate the order of relations placed at the end
```

Comment:
I think this JSDoc param should be removed

Reply:
good catch! Done :)